web/modules/custom/dpl_login/src/OpenIdUserInfoService.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Drupal\dpl_login;
+
+use Drupal\Core\Site\Settings;
+
+use function Safe\sprintf;
+
+/**
+ * Access Token.
+ */
+class OpenIdUserInfoService {
```

Comment:
Please update this DocBlock.

The current content does not make any sense to me.

Reply:
👍